### PR TITLE
chore(): expose `sendVectorToPlane`

### PR DESCRIPTION
**Pull request by ShaMan123**
## Motivation

send plane utils are proving to be valuble 
send a vector is also very common and completes the util

## Description

expose `sendVectorToPlane`
migrate send plane tests to jest, removing the `sendPointToPlane` old test that was unnecessary

### CODE REVIEW

File: src/util/misc/planeChange.spec.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    expect(calcPlaneChangeMatrix()).toEqual(iMatrix);
+    expect(calcPlaneChangeMatrix(undefined, m1)).toEqual(invertTransform(m1));
+    expect(calcPlaneChangeMatrix(iMatrix, m1)).toEqual(invertTransform(m1));
+    expect(calcPlaneChangeMatrix(m1, undefined)).toEqual(m1);
+    expect(calcPlaneChangeMatrix(m1, iMatrix)).toEqual(m1);
+    expect(calcPlaneChangeMatrix(m1, m1)).toEqual(iMatrix);
+    expect(calcPlaneChangeMatrix(m1, s)).toEqual(
+      multiplyTransformMatrices(invertTransform(s), m1)
+    );
+  });
```
Comment (ShaMan123) [inline src/util/misc/planeChange.spec.ts:32]: migrate from qunit

File: src/util/misc/planeChange.spec.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    const s: TMat2D = [2, 0, 0, 0.5, 50, 50];
+    const point = new Point(1, 1);
+    expect(sendVectorToPlane(point, s)).toEqual({ x: 2, y: 0.5 });
+    expect(sendVectorToPlane(point, undefined, s)).toEqual({ x: 0.5, y: 2 });
+    expect(
+      sendVectorToPlane(point, createRotateMatrix({ angle: 90 }), s)
+    ).toEqual({ x: -0.5, y: 2 });
+    expect(sendVectorToPlane(point, s, s)).toEqual(point);
+    expect(sendVectorToPlane(point)).toEqual(point);
+  });
```
Comment (ShaMan123) [inline src/util/misc/planeChange.spec.ts:56]: new test

File: src/util/misc/planeChange.spec.ts [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      )
+    );
+    expect(obj.calcOwnMatrix()).toEqualRoundedMatrix(
+      multiplyTransformMatrices(actual, m)
+    );
+
+    obj.group = group2;
+    expect(obj.calcTransformMatrix()).toEqualRoundedMatrix(
+      multiplyTransformMatrices(
+        multiplyTransformMatrices(group2.calcTransformMatrix(), actual),
```
Comment (ShaMan123) [inline src/util/misc/planeChange.spec.ts:91]: migrated from qunit

**Review by ShaMan123 — COMMENTED** [review]
this is also a v6 PR but can be merged after 6.0

**Review by asturur — APPROVED** [review]

**Review by asturur — APPROVED** [review]

**Pull request merged**